httpserver/server.go

```diff
@@ -36,18 +38,39 @@ const (
 	certName          = "tls.crt"
 	keyName           = "tls.key"
 	readHeaderTimeout = 5 * time.Second
+
+	// cacheTTL is the default time-to-live for the cache entry.
+	cacheTTL     = 30 * time.Second
```

Comment:
I'm wondering if we should expose this to the user and tie this value to be the same as the oras ttl cache that @binbin-li built.

Reply:
> I'm wondering if we should expose this to the user and tie this value to be the same as the oras ttl cache that @binbin-li built.

The TTL should be short enough to pick up verification result updates and not require a pod restart to invalidate cache. Given the goal for this cache is to reuse the results for concurrent requests, I don't think a long cache TTL is required.

I'll leave to you and other maintainers for the TTL time. Let me know what you think is appropriate based on the flow now and I can update it.

Reply:
+1 on making this exposed through the charts. Some customer might want to make sure the result is always "fresh". This property needs to be configurable to meet all customer scenario.

Reply:
Summary of community call: consensus was that it made sense to make this value tunable and exposed to the end user. We can set a default that most will not override but can be open to tuning if the use case requires it.